content/guides/installing-python/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Installing Python
+---
+
+Python is the external MobiFlight scripting language, e.g., needed to connect custom aircraft APIs to the WinWing CDUs.
+
+**Required Python environment (all three are checked by MobiFlight):**
```

Comment:
```suggestion
Required Python environment (all three are checked by MobiFlight):
```

Reply:
Bold isn't required. If it's meant to be a heading use a markdown heading level.